### DISCUSSION TITLE
How can I backup everything easily?

### DISCUSSION THREAD

**Discussion by davinhun** [YetiForceCompany/YetiForceCRM#16851] [How to] [answered]
Hello

I have installed YetCRM successfully and it's working great. I would like to backup the full installiation of this system, but I can not find documentation how to do it. 
First I would like to do a manual backup, then I can automate it via CRON.
I set BACKUP_PATH and I have access to the following module:
System tools / Backup Manager
  but this is empty, I can't do anything, I don't have any button to create a backup.

Or it would be enough to backup mySQL database frequently and I can restore it somehow?

Can you please help me?

**Comment by rskrzypczak** [comment]
The YetiForce application does not make any backups itself and does not have the functionality to restore data from backups.

The parameter you mentioned and the panel only give you the option of downloading files located in the indicated location directly from the application.

**Comment by vovpff** [comment, marked as answer]
You can manually make db dump and tar your CRM root. This is enough for backup
